prepare/prepare.go

```diff
@@ -56,3 +54,19 @@ func PreparePatient(patients models.PatientSlice) []map[string]interface{} {
 	}
 	return combinedData
 }
+
+func PrepareAdmin(admins models.AdminSlice) []map[string]interface{} {
```

Comment:
Is the return of this function of type `{ "admin_id": string; "user": Admin }`? If so, you should be able to define it and simplify the type used, instead of defining a map slice

Reply:
Adding to what murilo is saying, change your prepare functions to return a type instead of a map and use annotations when necessary so that these types can be encoded to jason. It will make it easier to read and understand what is the return in each endpoint.